frontend/packages/ux-editor/src/containers/FormContainer.tsx

```diff
@@ -44,10 +47,15 @@ export const FormContainer = ({
   const [expanded, setExpanded] = useState<boolean>(true);
 
   const handleDelete = useCallback((event: React.MouseEvent<HTMLButtonElement>): void => {
-    event.stopPropagation();
-    handleDeleteFormContainer(id);
-    handleDiscard();
-  }, [handleDeleteFormContainer, handleDiscard, id]);
+    if(ComponentType.Group ){   
+      const userConfirmed = window.confirm(t("ux_editor.component_confirm_delete"));
```

Comment:
Same here, I recommend to replace `window.confirm` with the `Popover` component

Reply:
done